### PR TITLE
Set correct group in rbac

### LINKED ISSUE #6879: Allow Resource with group set to v1 in rbac.ts are not showing, if your are not cluster admin
**Describe the bug**
Allow Resource with group set to v1 in rbac.ts are not showing, if your are not cluster admin

https://github.com/lensapp/lens/pull/6657/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#r1062429006

**To Reproduce**
1. Create a user than can only view a single namespace.
2. Open lens with this context
3. No ConfigMap is showing

**Expected behavior**
I can see the configmap....

**Environment (please complete the following information):**
- Lens Version: 6.3
- Installation method (e.g. snap or AppImage in Linux): OpenLens

### PR DESCRIPTION
**Pull request by farodin91**
Fixes #6879

Signed-off-by: Jan Jansen <jan.jansen@gdata.de>

### CODE REVIEW

**Review by Nokel81 — APPROVED** [review]

**Comment by Nokel81** [comment]
Thanks!

**Review by Nokel81 — CHANGES_REQUESTED** [review]
Actually there is slightly more work to do.

Can you also change the relevant uses of `di.inject(shouldShowResourceInjectionToken, { ...`?

For instance there is one in `src/common/front-end-routing/routes/cluster/config/config-maps/config-maps-route.injectable.ts` that should be updated too.

**Comment by farodin91** [comment]
@Nokel81  I hope caught all.

File: src/renderer/components/+workloads-overview/workloads/implementations/pods-workload.injectable.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ const podsWorkloadInjectable = getInjectable({
     return {
       resource: {
         apiName: "pods",
-        group: "v1",
+        group: ",
```
Comment (Nokel81) [inline src/renderer/components/+workloads-overview/workloads/implementations/pods-workload.injectable.ts]: missing a `"` I think

**Comment by Nokel81** [comment]
Also please sign your commit. Thanks for the fast response

**Comment by farodin91** [comment]
@Nokel81 done

**Review by Nokel81 — DISMISSED** [review]

**Comment by Nokel81** [comment]
Might have to fix the tests too

**Comment by farodin91** [comment]
i will try to fix it tomorrow.

**Comment by Nokel81** [comment]
I think it will be as simple as to just remove the `"v1"` in the tests too

**Review by Nokel81 — APPROVED** [review]

**Comment by farodin91** [comment]
@Nokel81 i have no idea why the missing tests are failing.

**Comment by Nokel81** [comment]
I can investigate

**Comment by Nokel81** [comment]
I noticed that this doesn't entirely solve the issue. Will look again on Monday.

**Comment by Nokel81** [comment]
@farodin91 if you run kubectl get --raw /api you should get back a JSON object that has a field called versions. What is its value? On GKE and minikube it is ["v1"]

**Comment by farodin91** [comment]
I get the same value.

**Pull request closed without merging**